src/stash-list/window.vue

```diff
@@ -161,9 +161,14 @@ export default defineComponent({
 
         async stash(ev: MouseEvent | KeyboardEvent) {this.attempt(async() => {
             if (this.visibleChildren.length === 0) return;
```

Comment:
One minor thing I caught here (and fixed during the merge)--we actually want to check `stashable_visible_children` directly, or we might still wind up creating an empty group (`createStashFolder()` below) even if there's nothing to stash.

Anyway, I took care of it, but wanted to mention in case you noticed the change during subsequent merges/rebases/etc.

Reply:
Oh, sorry. Thanks for noticing and fixing that issue!